docs/user_guide/freezing.rst

```diff
@@ -0,0 +1,29 @@
+Freezing ImageIO
+=======================
+
+ImageIO has wide-ranging platform support, and to achieve this we now lazy-load
+plugins. Most users only use a few plugins (1 or 2), so lazy-loading helps
+mitigate a combinatoric explosion of interactions. ImageIO's lazy loading is an
+interesting case as (1) these plugins are exactly the kind of thing that
+PyInstaller_ tries desperately to avoid collecting to minimize package size and
+(2) these plugins are super lightweight and won't bloat the package. 
+
+A simple workaround that is to add a command line switch when calling
+pyinstaller that will gather all plugins:
+
+.. code-block::
+
+  pyinstaller --collect-submodules imageio <entry_script.py>
+
+Alternatively, if you really want to limit the plugins used:
+
+.. code-block::
+
+  pyinstaller --hidden-import imageio.plugins.<plugin> <entry_script.py>
+
+Plugins that make use of external programs (like ffmpeg) will need to take
+extra steps to include necessary binaries_.
```

Comment:
```suggestion
In addition, some plugins (e.g., ffmpeg) make use of external programs, and for
these, you will need to take extra steps to also include necessary binaries_. If
you can't make it work, feel free to submit a `new issue
<https://github.com/imageio/imageio/issues>`_, so that we can see how to improve
this documentation further.
```

I think it's always nice to end on a friendly note if possible because I think it promotes engagement (though I don't have data on this) and it makes the library feel more approachable.